lib/resolvers/es.js

```diff
@@ -121,7 +131,10 @@ module.exports = {
 
     this.syncForwardDependency(pkg.name, destinationPath, tmpPath, mod, relativePath + this.extension);
     AllDependencies.add(descriptor, relativePath, mod.metadata.modules);
-    return AllDependencies.for(relativePath, pkg.name);
+
+    return AllDependencies.for(relativePath, pkg.name).filter(function(importName) {
```

Comment:
Some how files are being resolved multiple times even though I'm now filtering out things that have been synced. I have a feeling it's a matter of execution order.